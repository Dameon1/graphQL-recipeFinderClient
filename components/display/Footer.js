import React from 'react';
import '../styles/footer.css';

export default function Footer () {

  return (
    <div className='footer'>
      <p className="footerText">
        Powered by <a href="https://spoonacular.com/" className="footerLink" target="blank">@Spoonacular</a>
      </p>
    </div>
  )
};
