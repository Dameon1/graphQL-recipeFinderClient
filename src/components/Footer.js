//TODO ** CREATE STYLED COMPONENTS**

import React from "react";
import {FooterContainer,FooterText,FooterLink} from './styles';
import "./styles/footer.css";

export default function Footer() {
  return (
    <FooterContainer>
      <FooterText>
        Powered by{" "}
        <FooterLink
          href="https://spoonacular.com/"
          target="blank"
        >
          @Spoonacular
        </FooterLink>
      </FooterText>
    </FooterContainer>
  );
}