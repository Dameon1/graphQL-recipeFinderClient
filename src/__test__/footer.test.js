import React from "react";
import { shallow, mount } from "enzyme";
import Footer from "../components/display/footer";

describe("<Footer />", () => {
  it("Renders without crashing", () => {
    shallow(<Footer />);
  });
});
