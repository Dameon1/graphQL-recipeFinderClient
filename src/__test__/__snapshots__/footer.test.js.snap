// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Footer /> matches the snapshot 1`] = `
<styled.div>
  <styled.p>
    Powered by
     
    <styled.a
      href="https://spoonacular.com/"
      target="blank"
    >
      @Spoonacular
    </styled.a>
  </styled.p>
</styled.div>
`;
